.github/workflows/cmake.yml

```diff
@@ -0,0 +1,62 @@
+name: "CMake Release"
+
+on:
+  [push, workflow_dispatch]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+            - name: Windows Tiles Sounds x64 MSVC
+              preset: windows-tiles-sounds-x64-msvc
+              runs-on: windows-latest
+            - name: Windows Tiles Sounds x64
+              preset: windows-tiles-sounds-x64
+              runs-on: ubuntu-22.04
```

Comment:
I don't know if the cross compile is long for this world. I was hearing that someone had ideas to deprecate the cross build after 0.G releases.

Reply:
Good to know.